### PR TITLE
OSDOCS-12018-v2: fix FIPS links and update attribute 4.15-16

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16

Issue:
[OSDOCS-12018](https://issues.redhat.com/browse/OSDOCS-12018)

Link to docs preview:
[FIPS assembly](https://81967--ocpdocs-pr.netlify.app/microshift/latest/microshift_install/microshift-fips)

QE review:
- [x] QE has approved this change.

Additional information:
Goes with https://github.com/openshift/openshift-docs/pull/81965

### CODE REVIEW

File: microshift_install/microshift-fips.adoc [hunk 1]
```diff
@@ -16,8 +16,4 @@ include::modules/microshift-fips-rpm-system.adoc[leveloffset=+1]
 [id="additional-resources_microshift-fips_{context}"]
```
Comment (JoeAldinger) [inline microshift_install/microshift-fips.adoc:16]: I don't know if the order matters or not but there is [this](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies:~:text=Example%20unique%20anchor%20ID%20for%20an%20%22Additional%20resources%22%20title) in the contribution guide.

**Review by JoeAldinger — APPROVED** [review]
lgtm, one small question. Good PR.

File: modules/microshift-fips-rpm-system.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 Using FIPS with {microshift-short} requires enabling the cryptographic module self-checks in your {op-system-base-full} installation. After the host operating system has been configured to start with the FIPS modules, {microshift-short} containers are automatically enabled to run in FIPS mode.
 
-* When {op-system-base} is started in FIPS mode, {microshift-short} core components use the {op-system} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64 architectures.
+* When {op-system-base} is started in FIPS mode, {microshift-short} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 validation on only the x86_64 architectures.
```
Comment (dhensel-rh) [inline modules/microshift-fips-rpm-system.adoc:11]: Does FIPS mode only work on x86_64, and not on ARM ?

File: modules/microshift-install-system-requirements.adoc [hunk 3]
```diff
@@ -8,10 +8,10 @@
 
 The following conditions must be met prior to installing {microshift-short}:
 
-* A compatible version of {op-system} or {op-system-ostree}.
+* A compatible version of {op-system-base} or {op-system-ostree}.
 * AArch64 or x86_64 system architecture.
```
Comment (dhensel-rh) [inline modules/microshift-install-system-requirements.adoc:12]: The AArch64 looks funny.  Is there a reason the first two As are capitalized ?

File: modules/microshift-fips-rpm-system.adoc (hunk 2 above)
Comment (ShaunaDiaz) [reply modules/microshift-fips-rpm-system.adoc:11]: Just updating the attribute here, not reworking any content.

File: modules/microshift-install-system-requirements.adoc (hunk 3 above)
Comment (ShaunaDiaz) [reply modules/microshift-install-system-requirements.adoc:12]: https://redhat-documentation.github.io/supplementary-style-guide/#aarch64

**Pull request merged**